packages/wonder-blocks-button/components/button-core.md

```diff
@@ -0,0 +1,521 @@
+ButtonCores:
+```js
```

Comment:
I would break this up into multiple `js` blocks.  This will make the `view code` output more reasonable.